django_project/ford3/templates/campus_form.html

```diff
@@ -44,39 +49,38 @@ <h4 class="list-group-item-heading">
 
                         {% if wizard.steps.step1 == 4 %}
                             <div class='row'>
-                                <div class='col-md-5'>
-                                    <h5>All qualifications in {{ campus }} campus</h5>
+                                <div class='col-md-5 campus-all-qualifications'>
```

Comment:
I'm not sure about this. `col-md-5` already set a width. We don't need another custom class with a specific width of 437px.